src/utilities/file.service.ts

```diff
@@ -60,6 +63,54 @@ export class FileService {
         }
     }
 
+    async getNumberOfPdfPages(fileName: string): Promise<number> {
+        const retrievedFile = readFileSync(`resources\\pdfs\\${fileName}`);
+
+        return getDocument({ data: retrievedFile }).promise.then((document) => document.numPages);
+    }
+
+    async getConvertedPdfFile(fileName: string, pageNumber: number): Promise<StreamableFile> {
+        const pagesFolder = `${process.cwd()}\\out\\${fileName.replace('.pdf', '')}`;
```

Comment:
This would create a folder called out in addon directory? Why are you doing this?

Reply:
Was suppose to be so that we could group the output pages by filename in case we decided to cache them